Tests/JobsTests/JobsTests.swift

```diff
@@ -2,11 +2,20 @@ import XCTest
 import Jobs
 
 final class JobsTests: XCTestCase {
+<<<<<<< HEAD
```

Comment:
Looks like there are still some remainders of the merge conflict here. You can just delete this through line 12 and also line 20